quodlibet/_main.py

```diff
@@ -217,9 +268,10 @@ def init_plugins(no_plugins=False):
     print_d("Starting plugin manager")
 
     from quodlibet import plugins
+    # FIXME: This should probably go to `~/.local/lib/quodlibet`
     folders = [os.path.join(get_base_dir(), "ext", kind)
                for kind in PLUGIN_DIRS]
-    folders.append(os.path.join(get_user_dir(), "plugins"))
+    folders.append(os.path.join(get_config_dir(), "plugins"))
```

Comment:
What should be done here?

Reply:
I think this should still look in `.quodlibet/plugins` as that's where the long-standing users of this will still have them, but it's a bit contentious.

Ever since we merged plugins into the codebase here (and have gathered more and more), and running local builds is easier, it's tempting to argue not many people use this, but I have no data..

Reply:
Added the legacy folder as fallback.